<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportGcodeDialog</class>
 <widget class="QDialog" name="ExportGcodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export GCODE Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupFillStroke">
     <property name="title">
      <string>Laser Power and Speed</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutFillStroke">

      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutSpeed">
        <item>
         <widget class="QLabel" name="labelLaserSpeed">
          <property name="text">
           <string>Laser Speed:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="valueLaserSpeed">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>

      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutPower">
        <item>
         <widget class="QLabel" name="labelLaserPower">
          <property name="text">
           <string>Laser Power:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="valueLaserPower">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>

      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutMode">
        <item>
         <widget class="QLabel" name="labelLaserMode">
          <property name="text">
           <string>Laser Mode:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="valueLaserMode">
          <item>
           <property name="text">
             <string>Constant Power</string>
           </property>
          </item>
          <item>
           <property name="text">
             <string>Dynamic Power</string>
           </property>
          </item>
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>

      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutInitCode">
        <item>
         <widget class="QLabel" name="labelInitCode">
          <property name="text">
           <string>Init Code:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="valueInitCode">
         </widget>
        </item>
       </layout>
      </item>

      <item>
       <layout class="QHBoxLayout" name="horizontalExitCode">
        <item>
         <widget class="QLabel" name="labelExitCode">
          <property name="text">
           <string>Exit Code:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="valueExitCode">
         </widget>
        </item>
       </layout>
      </item>



     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDialogButtons">
     <item>
      <spacer name="horizontalSpacerDialogButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>ExportGcodeDialog</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>ExportGcodeDialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui> 
